interface/patient_file/front_payment.php

```diff
@@ -562,16 +561,16 @@ function toencounter(enc, datestr, topframe) {
 <title><?php echo xlt('Record Payment'); ?></title>
 
 <style>
- body    { font-family:sans-serif; font-size:10pt; font-weight:normal }
- .dehead { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:bold }
- .detail { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:normal }
-#ajax_div_patient {
-    position: absolute;
-    z-index:10;
-    background-color: #FBFDD0;
-    border: 1px solid #ccc;
-    padding: 10px;
-}
+    body    { font-family:sans-serif; font-size:10pt; font-weight:normal }
+    .dehead { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:bold }
+    .detail { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:normal }
```

Comment:
Hi @tywrenn 

Guess those classes can convert to the bootstrap class, except the font size, should I remove it?

Reply:
@stu01509 Yep you can go ahead and convert them all